aws-frauddetector-detector/src/aws_frauddetector_detector/helpers/api_helpers.py

```diff
@@ -565,6 +565,13 @@ def call_get_external_models(frauddetector_client, model_endpoint: str = None):
     return frauddetector_client.get_external_models(**args)
 
 
+@retry_not_found_exceptions
```

Comment:
We're not creating model versions in our resource providers currently, so we should not be retrying not found exceptions for get_model_version.

Reply:
Agreed. It should very well already exist.
Good catch, removing